Apps/WebClient/test/unit/Controllers/RobotsControllerTests.cs

```diff
@@ -33,58 +34,54 @@ public class RobotsControllerTests
         /// <summary>
         /// GetRobots - Custom Content.
         /// </summary>
-        [Fact]
-        public void ShouldGetRobotsCustom()
+        /// <param name="robotsFilePathExists">bool value indicating whether robots file path exists or not.</param>
+        /// <param name="robotsFileContentExists">bool value indicating whether robots file contents exist or not.</param>
+        /// <param name="invalidConfigRobotsFilePath">bool value indicating whether configuration robots file path is valid or not.</param>
+        [Theory]
+        [InlineData(true, true, null)]
+        [InlineData(true, false, null)]
+        [InlineData(false, null, null)]
+        [InlineData(true, true, true)]
+        public void ShouldGetRobots(bool robotsFilePathExists, bool? robotsFileContentExists, bool? invalidConfigRobotsFilePath)
```

Comment:
The nullable bools look like they could be made non-nullable without losing any information.

Reply:
Changed.